BugsAndExploits/ZeroTickFarms.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Zero tick farms
-description: How a zero tick farms work
+title: Zero Tick Farms
+description: How a Zero Tick Farms work
```

Comment:
is this singular or plural?

Reply:
yeah i agree and this is nitpicking but the title should probably be something like "Zero-Tick Farming"

Reply:
Done